app/controllers/signup.js

```diff
@@ -189,10 +188,15 @@ export default class SignupController extends Controller {
     // submit
     // https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#patch-usersself
     e.preventDefault();
+    let reqBody = this.formData;
+    if (!this.formData.website) {
+      const { website, ...body } = this.formData;
+      reqBody = body;
+    }
```

Comment:
Can we use the [delete operator ](ozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete)here instead?